assets/airports/kbos.json

```diff
@@ -1,713 +1,8666 @@
+    "RIPIT": ["N42.37965926440570", "W70.87753363878667"],
+    "ROBUC": ["N41.67888627271270", "W71.58510812769588"],
+    "RODEE": ["N42.59947479137811", "W69.94860296532975"],
+    "RUIZE": ["N41.47932773551621", "W72.20939708163479"],
+    "SCITU": ["N42.27885108087718", "W70.76041022851403"],
+    "SCOAR": ["N42.31261907489736", "W70.63753744075679"],
+    "SEETS": ["N42.64616012682112", "W72.31816172714302"],
+    "SEHAG": ["N42.46812389903683", "W70.78498298295322"],
+    "SEY"  : ["N41.16750149707092", "W71.57614511480660"],
+    "SHAAN": ["N42.57990100260009", "W70.82768318454781"],
+    "SHAUN": ["N42.67622768922109", "W70.64076911800313"],
+    "SOBAH": ["N42.23733435243515", "W70.14538641685365"],
+    "SOFEE": ["N41.84317584404800", "W71.25891488333254"],
+    "SPENO": ["N42.28015377867342", "W72.15408325582571"],
+    "SPYSD": ["N42.44969786960377", "W71.02708503898530"],
+    "SSAAM": ["N42.63223229311426", "W71.38600838661355"],
+    "SSOXS": ["N41.83679841823984", "W70.74610969645450"],
+    "STAYC": ["N42.50778089801794", "W70.79162971697076"],
+    "STEEM": ["N42.24206878950581", "W70.82566902434802"],
+    "STOLI": ["N42.74090505128394", "W73.11456067019694"],
+    "SUFFI": ["N43.02493494624861", "W69.91718206621232"],
+    "TAALE": ["N42.62623051498790", "W70.77189094701240"],
+    "TEKKK": ["N42.41271802017828", "W71.09863808543068"],
+    "TJAYY": ["N42.30499929589529", "W70.96041634171699"],
+    "TKMAN": ["N42.58190314953434", "W71.18086617558893"],
+    "TONNI": ["N42.37332190330239", "W70.61353832585554"],
+    "TTERI": ["N42.52960379229147", "W70.51528693755337"],
+    "VOCUS": ["N42.45408196815536", "W70.96192695116805"],
+    "WAATR": ["N42.45779707939299", "W70.82556831633800"],
+    "WACHU": ["N42.65238219377580", "W71.48107674805044"],
+    "WHATE": ["N42.43602344084837", "W72.76167980316850"],
+    "WIDUP": ["N42.48112286969136", "W71.16153022699130"],
+    "WIKID": ["N42.64087905393912", "W72.82418059636036"],
+    "WINNI": ["N42.11717719593455", "W71.12446967931068"],
+    "WINTA": ["N42.50347479630538", "W71.36667244869417"],
+    "WOONS": ["N41.95055393513426", "W71.50705718319130"],
+    "WYANE": ["N42.53042013761245", "W70.92476569547746"],
+    "WYLYY": ["N42.30081005100508", "W71.10067749414384"],
+    "WYMTH": ["N42.21224515672434", "W70.82714080131245"],
+    "YAARD": ["N42.22234583011669", "W70.83996955105606"],
+    "ZOMIB": ["N42.53546645852862", "W71.23494636628250"],
+    "ZUMOL": ["N42.40517069730649", "W70.98559333351842"]
+  },
+  "restricted": [
+    {
+      "name": "R-4102",
+      "height": "3995ft",
+      "coordinates": [
+        ["N42d31m15", "W71d36m28"],
+        ["N42d31m00", "W71d39m13"],
+        ["N42d30m45", "W71d39m13"],
+        ["N42d30m15", "W71d39m58"],
+        ["N42d29m45", "W71d41m13"],
+        ["N42d28m15", "W71d41m13"],
+        ["N42d28m00", "W71d39m43"],
+        ["N42d29m45", "W71d37m43"]
+      ]
+    }
+  ],
+  "runways":[
```

Comment:
Please define all runways with their smaller number first: `15L-33R`, `4R-22L`, `18-36`, etc. Then, the runways should be listed here ordered numerically (then by L-C-R) by the primary end:
```
4L-22R
4R-22L
9-27
14-32
15L-33R
15R-33L
```